cat package/bcm-refsw/20150326/disable-nxclient-linking-cube-application.patch
--- old/rockford/applications/khronos/v3d/nexus/cube/Makefile   2014-06-06 20:37:03.000000000 -0700
+++ new/rockford/applications/khronos/v3d/nexus/cube/Makefile   2014-10-28 12:22:44.806696019 -0700
@@ -84,6 +84,8 @@
 NXPL_LIB_TARGET := ${NEXUS_BIN_DIR}
 NXPL_OBJ_TARGET := ${B_REFSW_OBJ_ROOT}/rockford/middleware/platform/nexus/$(V3D_OBJ)

+USE_NXCLIENT = 0
+
 NEXUS_MODE ?= $(MODE)
 ifeq ($(NEXUS_MODE),)
 CFLAGS += -DSINGLE_PROCESS
